src/index.ts

```diff
@@ -64,12 +64,12 @@ export interface Config {
 
 interface QueryResultRow {
   lengths: string[]
-  values: string
+  values?: string
 }
 
 export interface Field {
   name: string
-  type: string
+  type?: string
```

Comment:
This is the part I'm not super thrilled about, since this type gets exposed publicly and is shared by the internal QueryResult. In practice, the Field that's used by QueryResult would have type optional, but the public Field we spit back out should have it required. I'm not familiar enough with TypeScript to know if there's a way to handle this better. I thought about duplicating the entire Field type as like, `interface RawField` and then keeping `export interface Field` for public, but it'd entirely be copy/pasted except for this one character difference. If there's a better way to handle that, I'd love to hear it.